lib/Sema/TypeCheckSwitchStmt.cpp

```diff
@@ -557,6 +559,8 @@ namespace {
         PAIRCASE (SpaceKind::Constructor, SpaceKind::Constructor): {
           // Optimization: If the heads of the constructors don't match then
           // the two are disjoint and their difference is the first space.
+
+          // FIXME: Exhuastiveness checking for labelled Enums
```

Comment:
Hold on: is this still not implemented properly?

Reply:
This has to do with extraneous warnings when there are labelled enums with the same base name.  There wasn't a good solution developed in the last pull request, and it was sort of out of the scope of the initial bug report, I thought it could be fixed in a different pull request.

Reply:
Ah, OK. That's usually called "redundancy checking". Had a small heart attack at the thought we had regressed the exhaustiveness checker.